internal/hash/checksum.go

```diff
@@ -203,6 +203,24 @@ func (c ChecksumType) String() string {
 	return "invalid"
 }
 
+// StringFull returns the type and all flags as a string.
+func (c ChecksumType) StringFull() string {
```

Comment:
is this needed? don't see it used

Reply:
I think its for printing purposes we can simply call it String() for stringer compatibility